compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -129,7 +129,7 @@ class Config(RepresentationMixin):
     amqp_port : int | None
         Port to use for AMQP connections. Note that only 5671, 5672, and 443 are
         supported by the Compute web services. If None, the port is assigned by the
-        services (typically 5671).
+        services (which default to 443).
```

Comment:
It would help future devs (or us) if there's a hint where this default is set. ie. 

"which default to 443 from the URL passed back, constructed from FUNCX_AMQP_URL in the services"

Update - Just saw Reid's comment, explicit assigning to 443 here does help clarity, but that would change code logic, and make setting :443 in the service URL redundant.  will discuss in # dev

Reply:
Decided against this since the docstrings are nominally user facing